test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/MSBuildTests.Solution.cs

```diff
@@ -11,28 +11,9 @@ public class MSBuildTests_Solution : AcceptanceTestBase
     private readonly AcceptanceFixture _acceptanceFixture;
     private const string AssetName = "MSTestProject";
 
-    static MSBuildTests_Solution()
-    {
-        string dotnetMuxerSDK = Directory.GetDirectories(Path.Combine(RootFinder.Find(), ".dotnet", "sdk")).OrderByDescending(x => x).First();
-        File.WriteAllText(Path.Combine(dotnetMuxerSDK, "Microsoft.Common.CrossTargeting.targets"), MicrosoftCommonCrossTargeting);
-        if (!File.Exists(Path.Combine(dotnetMuxerSDK, "Microsoft.Common.Test.targets")))
-        {
-            File.WriteAllText(Path.Combine(dotnetMuxerSDK, "Microsoft.Common.Test.targets"), MicrosoftCommonTesttargets);
-        }
-    }
-
     public MSBuildTests_Solution(ITestExecutionContext testExecutionContext, AcceptanceFixture acceptanceFixture)
         : base(testExecutionContext) => _acceptanceFixture = acceptanceFixture;
 
-    private void CheckPatch()
-    {
-        // https://github.com/dotnet/sdk/issues/37712
-        if (DateTime.UtcNow.Date > new DateTime(2024, 12, 1))
-        {
-            throw new InvalidOperationException("Check if we can remove the patch!");
-        }
```

Comment:
This is for sure failing now :)

I assume the code I deleted is what's being referred to here? @Evangelink @MarcoRossignoli 

In case the test didn't pass, I'll change the check to three months later to re-validate.

Reply:
yep let's try